### PR TITLE
fix(ci): fix problems with switch from fb to magma artifactory #12687

### PR DESCRIPTION
**Pull request by nstng**
Signed-off-by: Nils Semmelrock <nils.semmelrock@tngtech.com>

## Summary
Fixes changes introduced with #12687, #12688, #12689 and also #12799:
* 1st commit: fixes https://github.com/magma/magma/actions/workflows/docker-builder-devcontainer.yml
  * rebase merge problems resolved 
* 2nd commit: fixes "cwag deploy job" in https://github.com/magma/magma/actions/workflows/build_all.yml
  * magmacore artifactory not compatible to bionic base image -> updated to focal
  * **VORSICHT** had to remove ssl lib copy, as this does not exists in focal - is this essential?
  * `~/magma/cwf/gateway/docker$ docker-compose --file docker-compose.yaml --file docker-compose.override.yaml build sessiond`
* 3rd commit: fixes "agw container build" in https://github.com/magma/magma/actions/workflows/build_all.yml
  * some magma debs differ between fb and magma artifactory and are not compatible to an mme build
  * respective debs are fetched from the fb artifactory again
  * `~/magma/lte/gateway/docker$ docker-compose --file docker-compose.yaml --file docker-compose.override.yaml build gateway_c`
* 4th commit: fixes "agw container build" in https://github.com/magma/magma/actions/workflows/build_all.yml
  * **VORSICHT** had to remove packages bpfcc-tools and python3-bpfcc that are incompatible to magmas bcc-tools - are these essential?
  * after discussion with @m-govind this was changed to a conditional approach -> packages are only added on arm architecture
  * `~/magma/lte/gateway/docker$ docker-compose --file docker-compose.yaml --file docker-compose.override.yaml build gateway_python`

## Test Plan

CI
* `~/magma/cwf/gateway/docker$ docker-compose --file docker-compose.yaml --file docker-compose.override.yaml build sessiond`
* `~/magma/lte/gateway/docker$ docker-compose --file docker-compose.yaml --file docker-compose.override.yaml build gateway_c`
* `~/magma/lte/gateway/docker$ docker-compose --file docker-compose.yaml --file docker-compose.override.yaml build gateway_python`

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

File: lte/gateway/docker/services/python/Dockerfile [hunk 1]
```diff
@@ -104,8 +104,6 @@ RUN apt-get update && apt-get install -y \
   wget \
   ethtool \
   linux-headers-generic \
-  bpfcc-tools \
-  python3-bpfcc \
```
Comment (jheidbrink) [inline lte/gateway/docker/services/python/Dockerfile:108]: Given that the rest of this file wasn't changed, why is this change necessary?

Comment (jheidbrink) [reply lte/gateway/docker/services/python/Dockerfile:108]: Ah I see, just read it in the PR description.

Comment (jheidbrink) [reply lte/gateway/docker/services/python/Dockerfile:108]: @m-govind You originally added those lines in 6181da05 ([pull/11289](https://github.com/magma/magma/pull/11289)). Do you see a problem here?

**Review by tmdzk — APPROVED** [review]

**Comment by nstng** [comment]
In pairing with Govindarajan Mohandoss: added `bpfcc-tools` and `python3-bpfcc` to `lte/gateway/docker/services/python/Dockerfile` guarded by "${CPU_ARCH}" = "aarch64" as these packages are needed for arm builds.

**Review by jheidbrink — APPROVED** [review]
